Tiltfile

```diff
@@ -3,82 +3,101 @@
 def servantes():
   return composite_service([fe, vigoda, fortune, doggos, snack, hypothesizer, spoonerisms])
 
-def go_service(name, run_fn=None):
-  yaml = local_file('%s/deployments/%s.yaml' % (name, name))
+def fe():
+  yaml = read_file('servantes/deployments/servantes.yaml')
 
-  image_name = 'gcr.io/windmill-public-containers/servantes/%s' % name
+  image_name = 'gcr.io/windmill-public-containers/servantes/servantes'
 
-  img = build_docker_image('Dockerfile.go.base', image_name, '/go/bin/%s' % name)
-  path = '/go/src/github.com/windmilleng/servantes/%s' % name
+  img = build_docker_image('Dockerfile.go.base', image_name, '/go/bin/servantes')
+  path = '/go/src/github.com/windmilleng/servantes/servantes'
   repo = local_git_repo('.')
-  img.add(repo.path('./%s/' % name), path)
-
-  if run_fn:
-    run_fn(img)
+  img.add(repo.path('servantes/'), path)
 
-  img.run('go install github.com/windmilleng/servantes/%s' % name)
+  img.run('go install github.com/windmilleng/servantes/servantes')
   return k8s_service(yaml, img)
 
-def python_service(name, run_fn=None):
-  yaml = local_file('%s/deployments/%s.yaml' % (name, name))
+def vigoda():
+  yaml = read_file('vigoda/deployments/vigoda.yaml')
 
-  image_name = 'gcr.io/windmill-public-containers/servantes/%s' % name
+  image_name = 'gcr.io/windmill-public-containers/servantes/vigoda'
 
-  img = build_docker_image('Dockerfile.py.base', image_name, 'python /app/app.py')
+  img = build_docker_image('Dockerfile.go.base', image_name, '/go/bin/vigoda')
+  path = '/go/src/github.com/windmilleng/servantes/vigoda'
   repo = local_git_repo('.')
-  img.add(repo.path('./%s/' % name), "/app")
+  img.add(repo.path('vigoda/'), path)
```

Comment:
does it need to end in a slash? what happens if it doesn't?

Reply:
good question, I'll investigate.